assets/stylesheets/emergent-paywall.scss

```diff
@@ -0,0 +1,687 @@
+				}
+
+				// Error label below each field
+				.text-danger {
+					color: $alert-red;
+					margin: 0;
+					padding: 4px 0;
+				}
+
+			}
+
+			// Bottom block of text displayed when there are errors
+			.alert-warning {
+				margin-top:30px;
+
+				ul {
+					list-style: disc;
+					font-size: 12px;
+					padding: 10px;
+					margin-left: 20px;
+				}
+
+			}
+
+			// Separator after the form fields
+			.row-apart {
+				height:20px;
+			}
+
+		}
+
+		// Back link
+		a.btn-block {
+			float:right;
+			color: $blue-wordpress;
+			cursor: pointer;
+			margin-top: 5px;
+
+			&:hover {
+				color: mix( white, $blue-medium, 20 );
+			}
+		}
+
+		// Buy/continue/resend OTP button
+		button.btn-block, div[simple-template='link-password'] a.text-capitalize {
+			text-transform: capitalize;
+			background: $white;
+			border-color: $gray-lighten-20;
+			border-style: solid;
+			border-width: 1px 1px 2px;
+			color: $gray-dark;
+			cursor: pointer;
+			display: inline-block;
+			margin: 0;
+			outline: 0;
+			overflow: hidden;
+			font-weight: 500;
+			text-overflow: ellipsis;
+			text-decoration: none;
+			vertical-align: top;
+			box-sizing: border-box;
+			font-size: 14px;
+			line-height: 21px;
+			border-radius: 4px;
+			padding: 7px 14px 9px;
+			appearance: none;
+
+			&:hover {
+				border-color: $gray-lighten-10;
+			}
+
+			&.btn-primary {
+				background: $blue-medium;
+				border-color: #008ab3;
+				color: $white;
+
+				&:hover {
+					border-color: $blue-dark;
+				}
+			}
+		}
+
+		div[simple-template='link-password'] a.text-capitalize {
+			margin-top: 10px;
+		}
+
+		// Used to notify the user about enabling pop-ups
+		.alert-info {
+			margin-bottom: 20px;
+		}
+
+		// "Link your wallet"
+		h2 {
+			font-size: 20px;
+			font-weight: 300;
+		}
+
+	}
+
+
+	.content-alerts .alert {
+		border: 1px solid lighten( $gray, 25% );
+		font-size: 14px;
+		margin-right: 0;
+		border-radius: 8px;
+		background: $alert-yellow;
+		border-color: transparent;
+		color: $white;
+		a {
+			color: $white;
+		}
+	}
+
+	.footer {
+		margin-top: 10px;
+		background-color: $white;
+		border: none;
+		border-top: 1px solid $gray-light;
+
+		.footer-inner {
+			padding-bottom: 10px;
+			text-align: center;
+			font-size: 11px;
+
+			.logo {
+				display: none;
+			}
+		}
+	}
+}
+
+.busy-container {
+	text-align: center;
+	.busy-text {
+		display: none;
+	}
+}
+
+@keyframes rotate-spinner {
+	100% {
+		transform: rotate( 360deg );
+	}
+}
+
+div[ng-show='busyIndicator'] {
+	display: flex;
+	align-items: center;
+}
+
+.busy-indicator, .less-busy-indicator,  .less-busy-indicator:after, .busy-indicator:after {
+	display:block;
+	margin: auto;
+	box-sizing: border-box;
+	border: 0.1em solid transparent;
+	border-radius: 50%;
+	animation: 3s linear infinite;
+	animation-name: rotate-spinner;
+}
+
+.busy-indicator, .less-busy-indicator {
+	border-top-color: $blue-medium;
+	width: 35px;
+	height: 35px;
+}
+
+.busy-indicator:after, .less-busy-indicator:after  {
```

Comment:
Did stylelint complain about extra spaces and the other stuff? I see some gymnastics is required to overwrite the particularly bad markup, so no real drama.